### DISCUSSION TITLE
Api based interaction

### DISCUSSION THREAD

**Discussion by EduardoOliveira** [linuxserver/Heimdall#737] [Ideas]
Hi,

I am developing a container that detects other containers started with certain labels, and proxies their http services through the tor network.

I would like to use heimdall to keep track of the services ir launches, some of those services already exist within heimdall (transmission, sickchill, etc)

Is it possible, would it be possible, to create and delete apps within heimdall to keep track of the onion links?

Thanks

**Comment by banzo** [comment]
see also #428

**Comment by KodeStar** [comment]
This should be coming in v3 which is currently in development

**Comment by KodeStar** [comment]
I've started implementing this in 2.5.0, if you are still interested in the feature please let me know what you would like to see in it - https://github.com/linuxserver/Heimdall/projects/5
